standards/licencing-software-or-code.md

```diff
@@ -0,0 +1,58 @@
+---
+category: Building software
+expires: 2018-01-31
+---
+# Licensing software or code
+
+Everything we produce or use, whether open or closed, should have an
+appropriate licence applied (as [indicated by the service
+manual][service manual licencing]). Without a licence, others (even
+within MOJ) have no way of knowing for sure if they can use it.
+
+At the MOJ, we use the [MIT License](https://opensource.org/licenses/MIT).
+
+[service manual licencing]: https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable#licensing-your-code
+
+## Applying the MIT License to our work
```

Comment:
Shouldn't this be;

## Applying the MIT Licence to our work

? The code uses 'licence' elsewhere.

Reply:
It is correct to refer using License when talking about the MIT License specifically.